stdlib/InteractiveUtils/src/macros.jl

```diff
@@ -235,10 +235,10 @@ end
 macro time_imports(ex)
     quote
         try
-            Base.TIMING_IMPORTS[Threads.threadid()] += 1
+            Base.TIMING_IMPORTS[] += 1
```

Comment:
Does this get lowered to an atomic_inc?

Reply:
For the record, this was fixed in https://github.com/JuliaLang/julia/pull/41737